prymer/primer3/primer3.py

```diff
@@ -399,54 +380,38 @@ def design(self, design_input: Primer3Input) -> Primer3Result:  # noqa: C901
             **global_primer3_params,
             **design_input.to_input_tags(design_region=design_region),
         }
-        # Submit inputs to primer3
-        for tag, value in assembled_primer3_tags.items():
-            self._subprocess.stdin.write(f"{tag}={value}")
-            self._subprocess.stdin.write("\n")
-        self._subprocess.stdin.write("=\n")
-        self._subprocess.stdin.flush()
 
-        error_lines: list[str] = []  # list of errors as reported by primer3
-        primer3_results: dict[str, str] = {}  # key-value pairs of results reported by Primer3
+        # split the tags into sequence and non-sequence tags
+        seq_args = {}
+        global_args = {}
+        for tag, value in assembled_primer3_tags.items():
+            tag_str = f"{tag}"
+            if tag_str.startswith("SEQUENCE"):
+                seq_args[tag_str] = value
+            else:
+                global_args[tag] = value
```

Comment:
This works ... but I think we could probably also add a `is_global_arg()` to `Primer3InputTag` that does this more efficiently (e.g. based on ordinal) _or_ split `Primer3InputTag` into two classes - one for globals and one for sequence tags?

Reply:
Since `PrimerInputTag`'s value is a string, I don't think it's inefficient, so I am going to centralize this as properties in the enum, and get rid of the f-string call (since I don't think that is needed).